README.md

```diff
@@ -652,6 +653,51 @@ The supported Picture in Picture events on `PlayerView` are:
 
 Check [`events.ts`](https://github.com/bitmovin/bitmovin-player-react-native/blob/development/src/components/PlayerView/events.ts) for more information about them.
 
+### Customize HTML UI (Android and iOS only)
+
+The Bitmovin Player SDKs use the open source [Bitmovin Player Web UI](https://github.com/bitmovin/bitmovin-player-ui) on all platforms, except tvOS.
+This is cusomizable in multiple ways.
+
+#### Custom implementation
+
+Since the Bitmovin Player Web UI is open source, it can be forked and modified to tailor to any application's needs.
+See [Cusomizing the UI](https://github.com/bitmovin/bitmovin-player-ui#customizing-the-ui) section for details.
+
+In case a custom implementation of the Player UI is desired, configure the hosted JS and CSS files via the `StyleConfig`, like:
+
+```ts
+const player = usePlayer({
+  styleConfig: {
+    playerUiCss: 'CUSTOM_UI_CSS_URL',
```

Comment:
Nit:
As you add 3 sections (Custom implementation, CSS, Supplemental CSS) we could omit the `playerUiCss` here.

Reply:
My thinking was that if you fork and build your own UI most likely you would have your own CSS as well.